src/client/pages/AssessmentLanding/AssessmentLanding.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react'
+import { Navigate, Route, Routes, useParams } from 'react-router-dom'
+
+import { ClientRoutes } from '@meta/app'
+import { AssessmentName } from '@meta/assessment'
+
+import { useAppDispatch } from '@client/store'
+import { AssessmentActions, useCycle } from '@client/store/assessment'
+
+import CycleLanding from '../CycleLanding'
+
+const AssessmentLanding: React.FC = () => {
+  const dispatch = useAppDispatch()
+  const cycle = useCycle()
+
+  const { assessmentName, cycleName } = useParams<{ assessmentName: AssessmentName; cycleName: string }>()
+
+  useEffect(() => {
+    dispatch(AssessmentActions.initApp({ assessmentName, cycleName }))
```

Comment:
This shouldnt happen here

Reply:
Here we should fetch assessment if url param is different than assessment in store or if assessment in store isnundefined